### DISCUSSION TITLE
New tracker added

### DISCUSSION THREAD

**Discussion by FuNFooKaS** [ryelogheat/xpbot#37] [General]
Can we get another tracker added  to this project?

**Comment by ryelogheat** [comment]
Yeah, if you want me to do it then just send me an invite to the site at ryelogheat@pm.me or you could reference https://github.com/ryelogheat/xpbot/wiki/Tracker-Templates & try to do it yourself (it really isn't too bad :smiley: you can just copy blutopia.json & change some of the URLs & key/vals to your own then edit auto_upload.py to recognize your trackers acronym (e.g. blu, bhd, acm) )

**Reply by ryelogheat** [reply]
again if you'd just rather I do it, then send me an invite to the site at the email address I listed above ^^ & I'll see what I can do

**Reply by FuNFooKaS** [reply]
Sent

**Comment by ryelogheat** [comment]
I messaged you on your tracker about a few issues I have with your site & in its current shape I can't add it to xpbot. Sorry

**Comment by FuNFooKaS** [comment]
ah well  thanks for trying

**Comment by FuNFooKaS** [comment]
got er all fixed up now i believe if you would take a look when your free.
